modules/extension/transform/src/test/java/org/geotools/data/transform/TransformFeatureSourceTest.java

```diff
@@ -65,6 +65,35 @@ public void testInfo() throws Exception {
         assertEquals(new URI(schema.getName().getNamespaceURI()), info.getSchema());
     }
 
+    @Test
+    public void testGetDescription() throws Exception {
+        SimpleFeatureSource transformed = transformWithSelectionAndDescription();
+        assertEquals(
+                "the state name",
+                transformed
+                        .getSchema()
+                        .getDescriptor("state_name")
+                        .getType()
+                        .getDescription()
+                        .toString());
```

Comment:
Long method chain called 3 times. Maybe roll a ``getDescription(schema, attributeName)`` method to avoid ugly formatting?

Reply:
Fixed